packages/babel-parser/test/fixtures/typescript/type-arguments/new-false-positive-3/output.json

```diff
@@ -9,36 +9,40 @@
     "body": [
```

Comment:
I'm not sure if these changes are correct; I asked to the TS team.

Reply:
Yes, the updates are correct: https://www.typescriptlang.org/play?ts=4.8.0-dev.20220611#code%2FHYUw7gBAggPAQgPgFAGEg=

I'll rename the test.